lib/Backends/CPU/CPUDeviceManager.cpp

```diff
@@ -32,12 +32,15 @@ bool CPUDeviceManager::isMemoryAvailable(uint64_t estimate) {
 void CPUDeviceManager::addNetworkImpl(const Module *module,
                                       FunctionMapTy functions,
                                       ReadyCBTy readyCB) {
-  auto modIt = modules_.find(module);
-  if (modIt != modules_.end()) {
-    // Already have a module with this ID.
-    // TODO: should we replace it?
-    readyCB(module, ResultCode::Failed);
-    return;
+
+  std::set<std::string> &moduleFuncs = modules_[module];
```

Comment:
Does this check global name uniqueness or just within a module? What if there are two functions with the same name from different modules? Or is the expectation that is handled higher up the stack?

Reply:
You're right it should be functions_.